### DISCUSSION TITLE
Change the MENU to download the media

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#2044] [Ideas] [answered]
### Is your idea related to a problem?

It's not a problem, it's a suggestion.

### Describe your idea

**Suggestion 1**: Change the MENU to download /import the media:
1. Type dropdown menu
2. When clicking and choosing the type of media, go to the option: If it is jw.org, open the site / if it is chant, the option to choose appears / if it is other media, open the browser to choose the folder

**Suggestion 2**: Specify the types of media that M3 can download, for example, many people don't know that they can import / download a PDF file and M3 converts it to an image.

![image](https://github.com/sircharlo/meeting-media-manager/assets/90984283/839eaa7d-fd62-483b-bc05-e41f7cb69824)

### Describe alternatives you've considered

See the picture above

**Comment by mtdvlpr** [comment, marked as answer]
Outdated as of `v24.10.6`
